.github/ISSUE_TEMPLATE/bug_template.md

```diff
@@ -2,7 +2,7 @@
 name: Bug Report
 about: Report a bug encountered while operating Cilium
 title: ''
-labels: ''
+labels: 'kind/bug'
```

Comment:
Should we also set `kind/community-report` and `needs/triage`? I guess they should be set by default for "external contributors" and we'll now when to remove them on our own issues.

Reply:
Question is what is more common, "external" or "internal" reports. It might also be easy to forget to remove them, judging from how often others and myself forget to add a `release-notes/*` label :smile:

I'd be slightly tempted to leave it up to the people on triage duty to add `kind/community-report` and `needs/triage` where appropriate on "external" reports instead of requiring all developers to manually remove these labels.

Reply:
Ok, makes sense :+1: